include/text.php

```diff
@@ -1897,15 +1897,8 @@ function file_tag_save_file($uid, $item, $file)
 		intval($uid)
 	);
 	if (DBM::is_result($r)) {
-		if (! stristr($r[0]['file'],'[' . file_tag_encode($file) . ']')) {
-			q("UPDATE `item` SET `file` = '%s' WHERE `id` = %d AND `uid` = %d",
-				dbesc($r[0]['file'] . '[' . file_tag_encode($file) . ']'),
-				intval($item),
-				intval($uid)
-			);
-		}
-
-		Term::insertFromFileFieldByItemId($item);
+		$fields = ['file' => $r[0]['file'] . '[' . file_tag_encode($file) . ']'];
```

Comment:
Why did you remove the duplicate check?

Reply:
The old code did some duplicate check and had only executed the command under some circumstances - but the ```Term::insertFromFileFieldByItemId``` had been executed all the time. Since that call is now no more (it is executed in the item function), I preferred to do the call every time, so that the function call to ```Term::insertFromFileFieldByItemId``` is done every time as well.

Reply:
But this means a file can now be encoded multiple times in the `file` field, right?

Reply:
Yeah ... It's maybe better to not risk this.